Assets/Scripts/CreditScroller.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CreditScroller : MonoBehaviour {
```

Comment:
Class does nothing, so remove it!

Reply:
I created it since I'm going to be using it for the credits